test/behaviors/Crc.js

```diff
@@ -199,15 +252,15 @@ const deployUpgradeableCrc = async (CrcContract, admin, contractRegistry) => {
       admin,
     ],
   ];
-  const [, crc, proxy] = await deployUpgradeableContract(
+  const [, crc, proxy, , , versionName] = await deployLatestUpgradeableContract(
     artifacts,
     null,
-    CrcContract,
+    'CRC',
     contractRegistry,
     initParams
   );
 
-  return [participantRegistry, proxy, crc];
+  return [participantRegistry, proxy, crc, versionName];
```

Comment:
I just noticed that proxy and crc are switched from `deployLatestUpgradeableContract`. Is that intentional? For the future, I think you should absolutely avoid returning arrays and instead return objects with keys so that ordering mistakes are less likely to happen.

Reply:
I agree, I started the object switch with the newer functions and these should also be switched immediately, but need to fix associated tests. Yes I believe it was definitely intentional. Idr why